templates/deployment-streaming.yaml

```diff
@@ -62,14 +62,18 @@ spec:
             - name: streaming
               containerPort: {{ .Values.mastodon.streaming.port }}
               protocol: TCP
+          {{- if .Values.mastodon.streaming.livenessProbe }}
```

Comment:
Using `with` instead of `if` should reduce path duplication for the variable.

Reply:
Both would work due to the helm chart value defaults but since the rest of the file seems to adopt this convention for this reason, sure, why not :)

Reply:
resolved in latest commits